src/SFA.DAS.EAS.Account.Api.Types/ResourceViewModel.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace SFA.DAS.EAS.Account.Api.Types
+{
+    public class ResourceViewModel : IAccountResource
+    {
+        public string Id { get; set; }
+        public string Href { get; set; }
```

Comment:
Change Href to Uri (worth doing this for all occurrences as this is the correct term used with non HTML code for .net